internal/output/ghworkflow/gh_workflow.go

```diff
@@ -48,10 +48,10 @@ func NewOutput() *Output {
 			ciWorkflow: {
 				Name: "default",
 				// https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-a-fallback-value
-				Concurrency: Concurrency{
-					Group:            "${{ github.head_ref || github.run_id }}",
-					CancelInProgress: true,
-				},
+				// Concurrency: Concurrency{
```

Comment:
keeping commented since I found an edge case when the PR is initially opened it will get cancelled